### PR TITLE
fix: improve README.md

### LINKED ISSUE #480: Improve the Readme of the repository
The readme should explain what the tool is providing:
- features listing
- sample to try
- user guide
- how to contribute

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #480

### What does this PR do?

Improve the README. Draft PR as videos are missing

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#480 

### How to test this PR?

Read the content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,65 @@
...
+**Software:**
+
+- [Podman Desktop 1.8.0+](https://github.com/containers/podman-desktop)
+- [Podman 4.9.0+](https://github.com/containers/podman)
+
+**Hardware**
+
+LLMs AI models are heavy resource consumers both in terms of memory and CPU. Each of the provided models consumes about 4GiB of memory and requires at least 4 CPUs to run.
+
+So we recommend that no more that 3 simultaneous models running concurrently and a minimum of 12Gb of memory and at least 4 CPUs for the Podman machine. Please note that this is not relevant for WSL on Windows as the WSL technology the memory and CPU with the host desktop.
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply README.md:62]: I mentioned 12GB so what is your point ?

[the remark this answers is not in the document]
Comment (vrothberg) [reply README.md:62]: The 12GB are mentioned in the context of three simultaneously running models.  That with the paragraph above reads like 1 model = 4GB.  But for me, 8GB aren't enough for 1.

[the remark this answers is not in the document]
Comment (jeffmaury) [reply README.md]: Reworded in 5eb080f

File: README.md [hunk 2]
```diff
@@ -2,6 +2,65 @@
...
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Feedback](#feedback)
+
+
+
+## Technology
+
+The AI Studio extensions uses [Podman](https://podman.io) machines to run inference servers for LLM models and AI applications.
+The AI models can be downloaded, and common formats like [GGUF](https://github.com/ggerganov/ggml/blob/master/docs/gguf.md), [Pytorch](https://pytorch.org) or [Tensorflow](https://www.tensorflow.org).
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply README.md]: Fixed by 4bc841a

File: README.md [hunk 3]
```diff
@@ -2,6 +2,65 @@
...
+### Requirement 1. Software and hardware requirements
+
+**OS:**
+
+Compatible on Windows, macOS & Linux
+
+**Software:**
+
+- [Podman Desktop 1.8.0+](https://github.com/containers/podman-desktop)
+- [Podman 4.9.0+](https://github.com/containers/podman)
```
[the remark this answers is not in the document]
Comment (vrothberg) [reply README.md:56]: Makes sense, thanks!

**Review by vrothberg — APPROVED** [review]
LGTM

**Review by benoitf — APPROVED** [review]

**Pull request merged**